docs/index.html

```diff
@@ -42,9 +42,9 @@ <h1><code>wasm-pack</code></h1>
           <h2>📦✨ your favorite rust -> wasm workflow tool!</h2>
         </div>
         <div class="five columns" id="installer">
-          <a class="button button-primary" href="/wasm-pack/installer">✨ Install wasm-pack 0.6.0 ✨</a>
-          <p>15 Jan 2019 |
-            <a href="https://github.com/rustwasm/wasm-pack/releases/tag/v0.6.0">
+          <a class="button button-primary" href="/wasm-pack/installer">✨ Install wasm-pack 0.6.1 ✨</a>
+          <p>16 Jan 2019 |
```

Comment:
Be sure to update this when the release happens :blush:

Reply:
Yes! Good call!